### ISSUE TITLE
Panics if run out of proper context

### ISSUE THREAD

**Issue by stevedonovan** [Geal/cargo-external-doc#1]
```
$ cargo-external-doc
thread 'main' panicked at 'invalid JSON metadata: Syntax(EOFWhileParsingValue, 1, 0)', ../src/libcore/result.rs:799
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```
An opportunity for usage text ;) ?

**Comment by Geal** [comment]
probably. Can you send a PR? What caused the issue? Did you run it outside of a cargo project?

**Comment by stevedonovan** [comment]
Yes, outside of a cargo project. Fair enough! Thought that it could handle this a little more gracefully.

**Comment by Geal** [comment]
the readme states explicitely that I didn't put too much work into it 😉 
It's just a lot of `expect()` calls, but I could make some of them more readable.

**Comment by stevedonovan** [comment]
Sure, it's actually not a problem. Thought you'd appreciate a pedantic first issue.

Definitely something I'm going to use, since I'm a great believer in 'narrative documentation' alongside module docs.
